### PR TITLE
Use email field for password reset

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

**Review by pixeldesu — CHANGES_REQUESTED** [review]
I'm conflicted, because this _technically_ removes the ability to use either the screen_name or email for recovering the account.

Thinking about it, actually..._both_ and `screen_name` only are a bad choice, because these can be used to eventually spam someone. A screen name is public, so you can eventually go and send a ton of emails to someone.

It'd probably be a sensible choice to just switch password recovery to only email instead.

**Comment by raccube** [comment]
Just so we have it here:
image

File: app/views/devise/passwords/new.html.haml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   = bootstrap_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
     = devise_error_messages!
 
-    = f.text_field :login, autofocus: true, label: "User name"
+    = f.text_field :email, autofocus: true, label: "Email address"
```
Comment (nilsding) [inline app/views/devise/passwords/new.html.haml]: Hmm, wonder if this renders as an email field (with browser-side validation)

**Review by nilsding — APPROVED** [review]

File: app/views/devise/passwords/new.html.haml (hunk 1 above)
Comment (raccube) [reply app/views/devise/passwords/new.html.haml]: Now it does.

**Review by pixeldesu — APPROVED** [review]

**Pull request merged**